inspirehep/modules/workflows/views.py

```diff
@@ -288,30 +293,32 @@ def _parse_robotupload_result(result, workflow_id):
     already_pending_ones = WorkflowsPendingRecord.query.filter_by(
         record_id=recid,
     ).all()
-    if already_pending_ones:
-        current_app.logger.warning(
-            'The record %s was already found on the pending list.',
-            recid
-        )
-        response = {
-            'success': False,
-            'message': 'Recid %s already in pending list.' % recid,
-        }
-        return response
 
-    if not _is_an_update(workflow_id):
-        pending_entry = WorkflowsPendingRecord(
-            workflow_id=workflow_id,
-            record_id=recid,
-        )
-        db.session.add(pending_entry)
-        db.session.commit()
+    if _is_an_authors_workflow(workflow_id):
```

Comment:
Shall we move `already_pending_ones` inside the `if` (one query less)?

Reply:
:+1: